sdk-csharp-sample-net80/sdk-csharp-sample-net80.csproj

```diff
@@ -22,7 +22,7 @@
   <ItemGroup>
     <PackageReference Include="System.IO" Version="4.3.0" />
     <PackageReference Include="System.Threading" Version="4.3.0" />
-    <PackageReference Include="RestSharp" Version="109.0.0" />
+    <PackageReference Include="RestSharp" Version="112.0.0" />
```

Comment:
I think we may want to put this in it's own PR. The jump to 109 was chaotic and I think we may want to give customers the heads up about that and put it in its own release.

Reply:
ok, reverted